first_5_steps/1_install_tt_buda.md

```diff
@@ -105,10 +105,19 @@ python3 -m venv env
 source env/bin/activate
 ```
 
-#### Step 5. Pip install PyBuda and TVM
+#### Step 5. Upgade the pip installer 
 
 ```bash
-pip install pybuda-<version>.whl tvm-<version>.whl
+pip install --upgrade pip==24.0
+```
+The `pybuda-<version>.whl` file contains the PyBuda and `tvm-<version>.whl` file contains the latest TVM downloaded release(s).
```

Comment:
Remove this line. It's duplicate and fits under Step 6 below.

Reply:
done here https://github.com/tenstorrent/tt-buda-demos/pull/28/commits/fe6acb530ff775b4e471fd91cf5c0f4598ad2e42